packages/luna/src/components/Chat/ChatMessage/ChatMessage.stories.tsx

```diff
@@ -47,6 +47,34 @@ export const OwnerLongMessage: StoryObj<Component> = {
   },
 };
 
+export const PendingMessage: StoryObj<Component> = {
+  args: {
+    message: { ...messageTemplate, text: faker.lorem.words(10) },
+    owner: true,
+    pending: true,
+    retry: () => alert("retry sending"),
+  },
+};
+
+export const ErrorMessage: StoryObj<Component> = {
+  args: {
+    message: { ...messageTemplate, text: faker.lorem.words(10) },
+    owner: true,
+    error: true,
+    retry: () => alert("retry sending"),
+  },
+};
+
+export const RetryingMessage: StoryObj<Component> = {
```

Comment:
When the message is `pending` and in `error` state, it is unclear to the end user that the message in pending status. Please discuss it with Ziad.

Reply:
Discussed it today and he suggested to show it as regular pending state to not confuse the user @neuodev